routing/http/contentrouter/contentrouter_test.go

```diff
@@ -108,26 +128,40 @@ func TestFindProvidersAsync(t *testing.T) {
 
 	p1 := peer.ID("peer1")
 	p2 := peer.ID("peer2")
-	ais := []types.ProviderResponse{
-		&types.ReadBitswapProviderRecord{
-			Protocol: "transport-bitswap",
-			Schema:   types.SchemaBitswap,
-			ID:       &p1,
+	p3 := peer.ID("peer3")
+	p4 := peer.ID("peer4")
+	ais := []types.Record{
+		&types.PeerRecord{
+			Schema:    types.SchemaPeer,
+			ID:        &p1,
+			Protocols: []string{"transport-bitswap"},
 		},
-		&types.ReadBitswapProviderRecord{
-			Protocol: "transport-bitswap",
+		//lint:ignore SA1019 // ignore staticcheck
+		&types.BitswapRecord{
+			//lint:ignore SA1019 // ignore staticcheck
 			Schema:   types.SchemaBitswap,
 			ID:       &p2,
+			Protocol: "transport-bitswap",
+		},
+		&types.PeerRecord{
+			Schema:    types.SchemaPeer,
+			ID:        &p3,
+			Protocols: []string{"transport-bitswap"},
+		},
+		&types.PeerRecord{
+			Schema:    types.SchemaPeer,
+			ID:        &p4,
+			Protocols: []string{"transport-horse"},
 		},
-		&types.UnknownProviderRecord{
-			Protocol: "UNKNOWN",
+		&types.UnknownRecord{
+			Schema: "UNKNOWN",
```

Comment:
nit: cid.contact production uses lowercase ([example](https://cid.contact/routing/v1/providers/bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi)), lets do the same here
```suggestion
			Schema: "unknown",
```

Reply:
I changed it but it doesn't matter as anything other than bitswap has always been ignored: https://github.com/ipfs/boxo/pull/422#discussion_r1305340163